assets/js/state/sagas/databases.test.js

```diff
@@ -41,6 +43,53 @@ describe('SAP Systems sagas', () => {
     console.error.mockRestore();
   });
 
+  it('should update the absent_at field when the database instance is marked absent', async () => {
+    const { sap_system_id, instance_number, host_id, sid } =
+      databaseInstanceFactory.build();
+    const absent_at = Date.now();
```

Comment:
I bit strange this `absent_at` not going in the factory itself, isn't it?

Reply:
done